src/main/java/net/hypixel/nerdbot/curator/OldCurator.java

```diff
@@ -0,0 +1,419 @@
+package net.hypixel.nerdbot.curator;
+
+import lombok.Getter;
+import lombok.Setter;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageReaction;
+import net.dv8tion.jda.api.entities.channel.concrete.TextChannel;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.hypixel.nerdbot.NerdBotApp;
+import net.hypixel.nerdbot.api.channel.ChannelGroup;
+import net.hypixel.nerdbot.api.channel.ChannelManager;
+import net.hypixel.nerdbot.api.database.Database;
+import net.hypixel.nerdbot.api.database.DiscordUser;
+import net.hypixel.nerdbot.api.database.GreenlitMessage;
+import net.hypixel.nerdbot.util.Logger;
+import net.hypixel.nerdbot.util.Region;
+import net.hypixel.nerdbot.util.Time;
+import net.hypixel.nerdbot.util.Util;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.regex.Matcher;
+import java.util.stream.Collectors;
+
+@Getter
+@Setter
+public class OldCurator {
+
+    private final int limit;
+    private final List<ChannelGroup> groups;
+    private final List<GreenlitMessage> greenlitMessages;
+    private final List<DiscordUser> users;
+    private final boolean readOnly;
+    private Emoji agree, disagree, greenlit;
+    private long elapsed;
+
+    /**
+     * Initialize a new Curator object with a set limit and {@link ChannelGroup}
+     *
+     * @param limit  The amount of messages to curate
+     * @param groups The list of {@link ChannelGroup} to search through
+     */
+    public OldCurator(int limit, List<ChannelGroup> groups, boolean readOnly) {
+        this.limit = limit;
+        this.groups = groups;
+        this.readOnly = readOnly;
+
+        greenlitMessages = new ArrayList<>(limit);
+        users = new ArrayList<>();
+
+        agree = NerdBotApp.getBot().getJDA().getEmojiById(NerdBotApp.getBot().getConfig().getEmojis().getAgree());
+        disagree = NerdBotApp.getBot().getJDA().getEmojiById(NerdBotApp.getBot().getConfig().getEmojis().getDisagree());
+        greenlit = NerdBotApp.getBot().getJDA().getEmojiById(NerdBotApp.getBot().getConfig().getEmojis().getGreenlit());
+    }
```

Comment:
what happens in this initializer if we can't get one of the emojis?

Reply:
![image](https://user-images.githubusercontent.com/22944369/191503464-dd1a6e8d-c3f7-4015-bf33-26c50cb4d5d9.png)

I deleted this class and moved everything into `ChannelGroupCurator`